web/styles/app_components.css

```diff
@@ -1221,14 +1222,18 @@ div.overlay {
         align-items: center;
         box-sizing: border-box;
         width: 100%;
-        height: 100%;
-        margin: 1px;
+        height: 90%;
+        margin: 1.5px;
```

Comment:
Is there a reason to do half-pixel margins here? Those can't actually be rendered, but rounded up or down depending on the browser.

Reply:
Oh, wasn't sure that it get rounded. But makes sense.

We're no longer using margin in the updated code.